internal/datastreams/pathway.go

```diff
@@ -24,6 +24,9 @@ func isWellFormedEdgeTag(t string) bool {
 				return true
 			}
 		}
+		if t[:i] == "topic" && strings.Contains(t, "arn") {
```

Comment:
It might be a good idea to check that it is a valid arn (to the extent possible; if there isn't a convenient "is this an arn" function available you could get close by verifying that it matches one of the [three general formats](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html)):
```
arn:partition:service:region:account-id:resource-id
arn:partition:service:region:account-id:resource-type/resource-id
arn:partition:service:region:account-id:resource-type:resource-id
```
All of them begin with "arn:" and contain at least five colons, so checking for those two qualities should avoid most false positives like "carnival" or "arnold".